modin/core/storage_formats/pandas/query_compiler.py

```diff
@@ -2344,47 +2344,12 @@ def apply(self, func, axis, *args, **kwargs):
         # convert it to pandas
         args = try_cast_to_pandas(args)
         kwargs = try_cast_to_pandas(kwargs)
-        if isinstance(func, str):
-            return self._apply_text_func_elementwise(func, axis, *args, **kwargs)
```

Comment:
Do we need to keep `_apply_text_func_elementwise` now? It seems this is used nowhere.

Reply:
I see, you removed it.